packages/dd-trace/test/plugins/agent.js

```diff
@@ -60,7 +60,7 @@ module.exports = {
   },
 
   // Register a callback with expectations to be run on every agent call.
-  use (callback) {
+  use (callback, timeoutMs = 1000) {
```

Comment:
This used to have a second parameter which some tests might still be using, so this may not be safe.

Reply:
Fixed in fixup commit. Using an options object instead, so that we won't run into this.